app/uk/gov/hmrc/customs/notification/repo/NotificationWorkItemRepo.scala

```diff
@@ -28,6 +28,7 @@ import play.modules.reactivemongo.ReactiveMongoComponent
 import reactivemongo.api.indexes.{Index, IndexType}
```

Comment:
I have updated the sequence diagram for Circuit Breaker here to show that we do not wait for Push/Pull before we return 202 to MDG - please see https://confluence.tools.tax.service.gov.uk/display/CD/Push+Online+Retry+Spike
I realise you did not introduce the problem but I think it is probably cleaner to address this in your PR rather than raise another story.